client/src/components/img-button/img-button.js

```diff
@@ -0,0 +1,62 @@
+import Component from '@/components/component';
+
+import ImgBox from '@/components/img-box/img-box';
+import DefaultImg from '@/assets/images/button-default.svg';
+
+import styles from '@/styles/components/img-button/img-button.module.scss';
+
+const MAX_IMAGE_UPLOAD = 10;
+
+export default class ImgButton extends Component {
+  constructor(props) {
+    super(props);
+
+    this.$dom = this.createDom('div', {
+      className: styles['img-button'],
+    });
+
+    this.ImgBox = new ImgBox({
+      type: 'medium',
+      imageURL: '',
```

Comment:
```suggestion
      imageURL: this._props.imageURL || '',
```

이미지를 넣을 수 있는 방법이 없습니다.

Reply:
아이쿠.. 이미지를 안넣어줬었네요. 수정했습니다.